keybox/trusty_tipc_cmd.h

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2020 Intel Corporation
+ * Copyright 2020 The Android Open Source Project
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef _TRUSTY_TIPC_CMD_H_
+#define _TRUSTY_TIPC_CMD_H_
+
+#define PAGE_SIZE 4096
+#define KEYMASTER_PORT "com.android.trusty.keymaster"
+
+// Commands
+enum keymaster_command {
+    KEYMASTER_RESP_BIT = 1,
+    KEYMASTER_STOP_BIT = 2,
+    KEYMASTER_REQ_SHIFT = 2,
+
+    KM_GENERATE_KEY = (0 << KEYMASTER_REQ_SHIFT),
```

Comment:
i think for this tool, you can only define the cmd that the tool need, no need to list all of them.

Reply:
I would prefer Yang's version, since we can simply compare this file with keymaster header file to get the difference if we get any conflict.